UnitsNet/UnitDefinitions/MassMomentOfInertia.json

```diff
@@ -0,0 +1,83 @@
+﻿{
+  "Name": "MassMomentOfInertia",
+  "BaseUnit": "KilogramSquareMeter",
+  "XmlDoc": "A property of body reflects how its mass is distributed with regard to an axis.",
+  "Units": [
+    {
+      "SingularName": "GramSquareMeter",
+      "PluralName": "GramSquareMeters",
+      "FromUnitToBaseFunc": "x/1e3",
+      "FromBaseToUnitFunc": "x*1e3",
+      "Prefixes": [ "Milli", "Centi", "Kilo" ],
```

Comment:
Less properties makes intelli-sense work better/easier when developing so I would suggest to remove it. If someone needs it, it is a non braking change to add it later on.